projects/admin/src/app/routes/items-route.ts

```diff
@@ -174,6 +175,10 @@ export class ItemsRoute extends BaseRoute implements RouteInterface {
               {
                 label: _('Second call number'),
                 value: 'second_call_number'
+              },
+              {
+                label: _('Pending requests'),
```

Comment:
Preferably use the same label : "Current pending requests"

Reply:
If I put this wording in, it makes the sort menu presentation skip, because it is too long. That's why I chose this wording